tests/workflows/protocols/pw/test_base.py

```diff
@@ -23,7 +23,7 @@ def test_get_default_protocol():
 def test_default(fixture_code, generate_structure, data_regression, serialize_builder):
     """Test ``PwBaseWorkChain.get_builder_from_protocol`` for the default protocol."""
     code = fixture_code('quantumespresso.pw')
-    structure = generate_structure()
+    structure = generate_structure('silicon')
```

Comment:
I think it will be easier to understand the tests for someone that isn't familiar with them. They will immediately understand that a silicon structure is returned, and that it's possible to have other structures returned by the fixture as well.

Reply:
After all "Explicit is better than implicit" and "Readability counts". 🙃